server/routes/induction/update/checkYourAnswersUpdateController.ts

```diff
@@ -0,0 +1,47 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import createError from 'http-errors'
+import CheckYourAnswersController from '../common/checkYourAnswersController'
+import toCreateOrUpdateInductionDto from '../../../data/mappers/createOrUpdateInductionDtoMapper'
+import logger from '../../../../logger'
+import { InductionService } from '../../../services'
+
+/**
+ * Controller for the Check Your Answers screen after updating an Induction.
+ */
+export default class CheckYourAnswersUpdateController extends CheckYourAnswersController {
```

Comment:
I didn't have time to write a test for this (even though I would normally write one first! 😂 )

Reply:
I've added RR-766 to address this